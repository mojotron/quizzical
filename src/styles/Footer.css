.Footer {
  margin-top: 5rem;
  font-family: "Karla";
}
.Footer__info {
  color: var(--light-white);
  font-size: 1.3rem;
}
.Footer__info__link {
  font-weight: 600;
  color: var(--dark-white);
}
.Footer__info__link:hover,
.Footer__info__link:active {
  color: var(--dark-blue);
}
