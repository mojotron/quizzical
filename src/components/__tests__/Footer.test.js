import { render } from "@testing-library/react";
import Footer from "../Footer";

describe("Footer component", () => {
  test("render snapshot", () => {
    const { component } = render(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
