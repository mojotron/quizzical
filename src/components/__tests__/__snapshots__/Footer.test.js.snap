// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component render snapshot 1`] = `undefined`;
